site/app/models/SimpleStat.php

```diff
@@ -54,6 +54,8 @@ class SimpleStat extends AbstractModel {
     public function __construct(Core $core, $details = []) {
         parent::__construct($core);
         if (isset($details['gc_id'])) {
+            $bad_submissions_cookie = array_key_exists('include_bad_submissions', $_COOKIE) ? $_COOKIE['include_bad_submissions'] : 'omit';
+            $null_section_cookie = array_key_exists('include_null_section', $_COOKIE) ? $_COOKIE['include_null_section'] : 'omit';
```

Comment:
This is a great place to use the [null coalescing](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) operator.

```suggestion
            $bad_submissions_cookie = $_COOKIE['include_bad_submissions'] ?? 'omit';
            $null_section_cookie = $_COOKIE['include_null_section'] ?? 'omit';
```

Reply:
I love that operator! A good feature of PHP.